libs/vm/src/object.cpp

```diff
@@ -116,143 +116,143 @@ std::size_t Object::GetHashCode()
 
 bool Object::IsEqual(Ptr<Object> const & /* lhso */, Ptr<Object> const & /* rhso */)
 {
-  RuntimeError("IsEqual operator not implemented");
+  RuntimeError(std::string(__func__) + "operator not implemented");
```

Comment:
Is `__func__` portable? I am sure that we have a system independent macro somewhere for this?

Reply:
Yes, it is portable, because of C99: https://isocpp.org/wiki/faq/cpp11-language-misc#cpp11-c99

Reply:
Yep, I know what should happen but I would suggest that almost certainly MSVC will not support this for instance. In any case, it makes sense to me to make a fetch specific macro for this so that when we run into this case, it is a simple update to a single header file.

Reply:
VS 2015 or later supports __func__ for C++11+ code.

Reply:
We can add such macro when it will be really needed. All relevant C++ compilers I know of do support `__func__`  (it has been made part of C++11 standard after all).